apps/linode-marketplace-openlitespeed-django/roles/common/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+# ufw_install
```

Comment:
Does openlightspeed include a firewall UI tool? if so we can remove ufw. If they don't then we can keep ufw and add the suggested ports to the ufw_rules.yml

Reply:
Yes. I meant to remove this! :)